docs/user-guide/configuring.md

```diff
@@ -509,7 +509,9 @@ your-project
 
 If there is an `.eslintrc` and a `package.json` file found in the same directory, `.eslintrc` will take a priority and `package.json` file will not be used.
 
-**Note:** If you have a personal configuration file in your home directory (`~/.eslintrc`), it will only be used if no other configuration files are found. Since a personal configuration would apply to everything inside of a user's directory, including third-party code, this could cause problems when running ESLint.
+**Note:** If you have a personal configuration file in your home directory `~/.eslintrc`), it will only be used if no other configuration files are found. Since a personal configuration would apply to everything inside of a user's directory, including third-party code, this could cause problems when running ESLint.
```

Comment:
I'm not sure this paragraph is needed anymore, if you think that it does, second sentence needs rework, since it's hard to understand that it's trying to explain why home directory is a special case, and instead it reads more like an instruction.

Reply:
No I agree with you. I think that as I was reading the doc and the issue, the only reconciliation I could see that would help clear the issue could be the placement of the note. I don't feel strongly about it, so we can keep the documentation as is. thanks!